viziphant/unitary_event_analysis.py

```diff
@@ -69,91 +69,61 @@ def plot_ue(data, Js_dict, sig_level, binsize, winsize, winstep,
 
     Parameters
     ----------
-    data : list of list of neo.SpikeTrain
+    spiketrains : list of list of neo.SpikeTrain
         A nested list of trials, neurons and their neo.SpikeTrain objects,
         respectively. This should be identical to the one used to generate
         Js_dict.
     Js_dict : dict
-        The output of elephant.unitary_event_analysis.jointJ_window_analysis
+        The output of
+        :func:`elephant.unitary_event_analysis.jointJ_window_analysis`
         function. The values of each key has the shape of:
 
-        * different pattern hash --> 0-axis;
-        * different window --> 1-axis.
+          * different window --> 0-axis.
+          * different pattern hash --> 1-axis;
 
         Dictionary keys:
 
-        * 'Js': list of float
-
+        'Js': list of float
           JointSurprise of different given patterns within each window.
-        * 'indices': list of list of int
-
+        'indices': list of list of int
           A list of indices of pattern within each window.
-        * 'n_emp': list of int
-
+        'n_emp': list of int
           The empirical number of each observed pattern.
-        * 'n_exp': list of float
-
+        'n_exp': list of float
           The expected number of each pattern.
-        * 'rate_avg': list of float
-
+        'rate_avg': list of float
           The average firing rate of each neuron.
-    sig_level : float
+
+    significance_level : float
         The significance threshold used to determine which coincident events
         are classified as unitary events within a window.
-    binsize : quantities.Quantity
-        The size of bins for discretizing spike trains. This value should be
-        identical to the one used to generate Js_dict.
-    winsize : quantities.Quantity
-        The size of the window of analysis. This value should be identical to
-        the one used to generate Js_dict.
-    winstep : quantities.Quantity
-        The size of the window step. This value should be identical to the one
-        used to generate Js_dict.
-    pattern_hash : list of int
-        List of interested patterns in hash values. This value should be
-        identical to the one used to generate Js_dict.
-    plot_params_user : dict
-        A dictionary of plotting parameters used to update the default plotting
-        parameter values. Dictionary keys:
-
-        * 'events' : list
+    **plot_params
+        User-defined plotting parameters used to update the default plotting
+        parameter values. The valid keys:
 
+        'events' : list
```

Comment:
```suggestion
        'events' : dict
```
See `plot_params_default` and L255-258